flutter/lib/models/model.dart

```diff
@@ -1824,12 +1833,25 @@ class CanvasModel with ChangeNotifier {
     _lastViewStyle = viewStyle;
     _scale = viewStyle.scale;
 
+    // Apply custom scale percent when in Custom mode
+    if (style == kRemoteViewStyleCustom) {
+      try {
+        _scale = await getSessionCustomScale(sessionId);
+      } catch (e, stack) {
+        print('Error in getSessionCustomScale: \$e\n\$stack');
+        _scale = 1.0;
+      }
+    }
+
     _devicePixelRatio = ui.window.devicePixelRatio;
     if (kIgnoreDpi && style == kRemoteViewStyleOriginal) {
       _scale = 1.0 / _devicePixelRatio;
     }
     _resetCanvasOffset(displayWidth, displayHeight);
-    _imageOverflow.value = _x < 0 || y < 0;
+    final overflow = _x < 0 || y < 0;
+    if (_imageOverflow.value != overflow) {
```

Comment:
There is actually no need to do this check.

https://github.com/jonataslaw/getx/blob/dae1f77021149615ed756bc9b26ee4c961f460d9/lib/get_rx/src/rx_types/rx_core/rx_impl.dart#L101


It's Ok though, there is no problem with this check.